docs/source/developers/guide/step_by_step/finding_issues.rst

```diff
@@ -29,4 +29,82 @@
 
 ****************************
 Finding good first issues 🔎
-****************************
\ No newline at end of file
+****************************
+
+You have successfully built Arrow library, congrats!
+
+The next step is finding something to work on. As was mentioned before,
+you might already have a bug to fix in mind, or a new feature that you want to implement.
+Or you still need an issue to work on and you need some help with finding one.
+
+For both cases JIRA is the issue tracker we use.
+
+First we will explain how to use JIRA if you have a fix or a feature to work on but
+there is no issue about it created. In that case you need to create an issue yourself.
+
+Secondly we will show a way to find good first issues.
+
+How to create a JIRA issue
+==========================
+
+First thing you need to do is to make an account on the ASF JIRA following
+`this link <https://issues.apache.org/jira/secure/Signup!default.jspa>`_. You will have to
+select your language and choose an avatar. If the registration is successful you will see:
+
+.. figure:: jira_1.jpeg
```

Comment:
Instead of using numbers, I think it'll be easier to find these back using distinctive names, e.g. `jira_new_account.jpeg`.